### PR TITLE
Create EA.md

### PR DESCRIPTION
**Pull request by monicazng**
REVIEW before merging

### CODE REVIEW

File: docs/jobs/openings/EA.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+## Executive Assistant opening @ Aragon (Zug, Switzerland)
+
+Aragon is an open source, decentralized project [fighting for the freedom](https://www.youtube.com/watch?v=AqjIWmiAidw) to organize. We believe that decentralized organizations can solve the world's worst problems.
```
Comment (luisivan) [inline docs/jobs/openings/EA.md]: > Aragon is an open source, decentralized project [fighting for the freedom](https://www.youtube.com/watch?v=AqjIWmiAidw) to organize.

I guess there should be something after _to organize_?

File: docs/jobs/openings/exec-assistant.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+## Executive Assistant opening @ Aragon (Zug, Switzerland)
+
+Aragon is an open source, decentralized project [fighting for the freedom](https://www.youtube.com/watch?v=AqjIWmiAidw) to organize. We believe that decentralized organizations can solve the world's worst problems.
+
+We’re looking for an Executive Assistant to support the Aragon founders with organization, coordination and time optimization. The ideal candidate must not only be someone highly efficient, with love for time self-management and adaptive to fast-paced environments, but also curious and eager to learn about decentralization.
```
Comment (Smokyish) [inline docs/jobs/openings/exec-assistant.md]: > eager to learn about decentralization.

How about `eager to learn about the decentralization and open-source movements.` ?

File: docs/jobs/openings/exec-assistant.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+* Facilitate meetings by structuring agendas, taking notes and following up.
+* Prepare and keep track of expense reports, invoices and cashflows.
+
+### Who you are
+* You put attention to details, organization, and process.
+* You have the ability to anticipate needs and find effective solutions with common sense.
+* You have a high sense to protect and respect privacy.
+* You have some knowledge about the crypto space and you’re excited to learn more.
+* You have excellent written and spoken English communication skills.
+* You’re comfortable working with online tools for reporting and project, calendar, inbox and password management, among others.
```
Comment (Smokyish) [inline docs/jobs/openings/exec-assistant.md]: > among others

Should be `among other things` i think

File: docs/jobs/openings/exec-assistant.md (hunk 2 above)
Comment (monicazng) [reply docs/jobs/openings/exec-assistant.md]: I think decentralization implicitly involves open source. So I'd leave it as it is.

File: docs/jobs/openings/EA.md (hunk 1 above)
Comment (monicazng) [reply docs/jobs/openings/EA.md]: All the openings have the same line, I think the source is our website actually. What would you add? Maybe `fighting for the freedom of organization`?

Comment (luisivan) [reply docs/jobs/openings/EA.md]: My bad, LGTM

**Pull request merged**